### ISSUE TITLE
French aspell issue

### ISSUE THREAD

**Issue by strikersh** [void-linux/void-packages#16659]
# System
* xuname:  
Void 5.3.11_1 x86_64 GenuineIntel notuptodate rrrmFFFFFF
* package:  
aspell-fr-0.50.3_2
aspell-0.60.8_1

### Expected behavior
start the TUI showing errors in text
### Actual behavior
```
$ aspell -c -l fr ~/test.txt
Erreur : The file "/usr/lib/aspell-0.60/fr-40-only.rws" is not in the proper format.
```
### Steps to reproduce the behavior
just try to use aspell in french.
when writing this issues I think I found why there is an issue the fr package seams to use an outdated version of aspell

**Comment by strikersh** [comment]
Ok so the dictionary isn't outdated on gnu ftp the file was created in 2004 and wasn't updated since

**Comment by flexibeast** [comment]
Unfortunately, this is a [known issue with Aspell](http://aspell.net/man-html/Using-32_002dBit-Dictionaries-on-a-64_002dBit-System.html).

If anyone has any ideas about how we might be able to deal with this issue in the relevant Void packages, i'm willing to work on implementing them.

**Comment by mvf** [comment]
@flexibeast To fix this, I'd suggest moving the dictionary compilation to install time, similar to the bytecode compilation step when installing python modules. This ensures compatibility and still keeps dictionaries `noarch`. It might require some clever preprocessing at build time to keep dependencies down. I don't think we want `aspell-fr` etc. to depend on `make`, for instance.

@strikersh As a workaround, rebuilding `aspell-fr` fixes it for me.

**Comment by flexibeast** [comment]
Hmm .... So the `template` file should manually replicate what running `./configure && make` would otherwise do?

**Comment by Eluminae** [comment]
This issue still persist here. I run the local rebuild but is it possible to fix this definitely ?

**Comment by ericonr** [comment]
Ok, this was fixed in #24248 by actually shipping dictionaries compiled for x86_64.

A mirror of this issue now exists on `armv*`, unfortunately.
